pagopa/io-app
# chore: [IOBP-992] Update analytics body sent to mixpanel for psp not available

## Short description
This pull request improve the tracking of payment analytics data.

## List of changes proposed in this pull request
- Reordered and added fields such as `attempt`, `amount`, and `expiration_date` to the `trackPaymentsPspNotAvailableSelectNew` function
- Added fields `payment_method_selected` and `amount` to the analytics data, and reordered existing fields

## How to test
- Try to pay something (with a psp 404 error from dev server)
- Check if mixpanel message are sent correctly
- Try to change payment method from the CTA `Scegli un altro metodo`
- Check if mixpanel message are sent correctly
